src/vl-wizard-pane.js

```diff
@@ -169,12 +168,6 @@ export class VlWizardPane extends VlElement(HTMLElement) {
         this.toggleAttribute('data-vl-previous-pane-disabled', value);
     }
 
-    _processTitle() {
-        if (this._titleSlot) {
-            this._titleSlot.setAttribute('data-vl-wizard-focus', '');
```

Comment:
Dit komt dan wel precies niet meer overeen met de demo van webuniversum?

Reply:
Klopt, details kan je nalezen in het gekoppeld issue #21.